src/Avalonia.Visuals/Media/PixelPoint.cs

```diff
@@ -252,4 +252,12 @@ public override string ToString()
             return string.Format(CultureInfo.InvariantCulture, "{0}, {1}", X, Y);
         }
     }
+
+    public static class PixelPointHelper
```

Comment:
There's already the static `PixelPoint.FromPoint` method which does this.

Reply:
I just removed the method and switched the code to using PixelPoint.FromPoint(...) method instead.

Someone needs to test the change on Mac - I am temporarily out of Mac. I tested it on Windows with USE_MANAGED_DRAG set to true.